DESCRIPTION

```diff
@@ -75,7 +75,7 @@ Suggests:
     forcats,
```

Comment:
Need to add a version requirement for `  dplyr (>= 1.0.0)`

Reply:
1.0.0 isn't actually required

Reply:
The comment is for `dplyr`. I can't comment up there

Reply:
I know, I don't want to require dplyr 1.0, it should work with older versions too